wgpu-types/src/lib.rs

```diff
@@ -1063,7 +1063,7 @@ bitflags::bitflags! {
 
         /// Supports samplers with anisotropic filtering. Note this isn't actually required by
         /// WebGPU, the implementation is allowed to completely ignore aniso clamp. This flag is
-        /// here for native backends so they can comunicate to the user of aniso is enabled.
+        /// here for native backends so they can communi    cate to the user of aniso is enabled.
```

Comment:
```suggestion
        /// here for native backends so they can communicate to the user of aniso is enabled.
```

Reply:
I keep on messing up this pr Oo

Reply:
My cat has tried to... contribute to PRs before :)

Reply:
... wow. okay. Yeah is late here and I shouldn't be let on a computer either. Read the diff the other way round xD